openassessment/xblock/static/js/src/lms/oa_course_items_listing.js

```diff
@@ -28,9 +33,21 @@ export class CourseItemsListingView {
         const hasAssessmentType = this.model.get(this.type ? this.type : 'staff_assessment');
         let link = null;
         if (itemViewEnabled && (!this.type || (this.type && hasAssessmentType))) {
+          const displayValue = this.title || formattedValue;
+          if (esgEnabled) {
+            const id = this.model.get('id');
+            link = $('<a>', {
+              text: displayValue,
+              title: displayValue,
+              href: `${esgUrl}/${id}`,
+              class: 'staff-esg-link',
+            });
+            this.$el.append(link);
+            return this;
+          }
           link = $('<a>', {
-            text: formattedValue,
-            title: this.title || formattedValue,
+            text: displayValue,
+            title: displayValue,
```

Comment:
⚠️ This looks like a change in behavior. Is this intentional?

Reply:
Tested there isn't any thing change. Title doesn't get define except the additional column I add.